justspacesapp/templates/survey_list.html

```diff
@@ -27,6 +28,7 @@ <h2 class="card-title">Run Survey</h2>
           <td>{{survey.updated}}</td>
           <td><a href="{% url 'fobi.edit_form_entry' survey.id %}"><i class="fas fa-pencil-alt icon edit"></i></a></td>
           <td><a href="{% url 'fobi.view_form_entry' survey.slug %}"><i class="fas fa-clipboard-list icon run"></i></a></td>
+          <td><a href="{% url 'fobi.delete_form_entry' survey.id %}"><i class="fas fa-times icon delete"></i></a></td>
```

Comment:
Unrelated to this line of code! But could we move partials into a partials dir, e.g., `templates/partials/sidebar.html`? Let's do that in a separate PR. I'll open an issue.

Reply:
I saw you opened #62, thank you! I'm not sure why I didn't do this to start.